web/index.html

```diff
@@ -1435,18 +1435,29 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 		colors: '#FF5588',
 		heads: [
 			gaugeChart('Utilization', '12%', '', '#FF5588')
-		]
+		],
+		info: 'Percent of time the disk was doing something. Read from field 10.'
```

Comment:
I suggest this text:

Disk Utilization measures the amount of time the disk was busy with something. This is not related to its performance. 100% means that the Linux kernel always had an outstanding operation on the disk. Keep in mind that depending on the underlying technology of the disk, 100% here may or may not be an indication of congestion.